types/sdk.js

```diff
@@ -60,7 +60,7 @@ export type AuthMiddlewareOptions = {
   scopes: Array<string>;
 }
 export type HttpMiddlewareOptions = {
-  host?: string;
+  host: string;
```

Comment:
Alright, let's do it this way then. Can you change it also for the other middleware option types?